### PR TITLE
295 jay deletes

### PR DESCRIPTION
**Pull request by seaparks**
@Preetam can you eyeball this real quick for me?

### CODE REVIEW

File: content/getting-started/supported-systems.md [hunk 1]
```diff
@@ -1,93 +1,92 @@
...
-- CentOS/RHEL 5.x is not supported by Go, although we've had some
-success with more recent kernels (2.6.18-164 at least, but with a performance
-penalty up to 2.6.28; see
-[this](http://dave.cheney.net/2013/06/18/how-to-install-go-1-1-on-centos-5)
-article for more information).
-- FreeBSD is supported for all non-EOL versions, older versions are untested. See
-[here](https://www.freebsd.org/security/security.html#sup) for a list of supported versions.
+---
+title: Supported Systems
+maintitle: Supported Systems
```
Comment (Preetam) [inline content/getting-started/supported-systems.md]: Invalid characters?

File: content/getting-started/supported-systems.md [hunk 2]
```diff
@@ -1,93 +1,92 @@
...
+We support Amazon RDS for MySQL by using an alternative method to capture queries.
+Instead of sniffing a TCP socket, we can capture performance statistics from the
+`PERFORMANCE_SCHEMA` tables, provided you're using MySQL version 5.6.14 or newer.
+Note that the `statements_digest` consumer (table `setup_consumers`) needs to be
+enabled for this to work and, if you want samples to be captured, the
+`events_statements_history_long` consumer has to be enabled as well. Note also
+that this approach is not exclusive to Amazon RDS. We can use it for any instance
+where sniffing is not an option, like off-host installations and TCP connections using SSL.
+
+We also support Amazon Aurora, and the same `PERFORMANCE_SCHEMA` instructions
```
Comment (Preetam) [inline content/getting-started/supported-systems.md]: "Aurora for MySQL" ?

File: content/getting-started/supported-systems.md [hunk 3]
```diff
@@ -1,93 +1,92 @@
...
+
+Note that the `PG_STAT_STATEMENTS` approach is not exclusive to Amazon RDS. We can
+actually use it for any instance where sniffing is not an option, like off-host 
+installations and TCP connections using SSL.
+
+## Versions of Redis
+
+VividCortex supports Redis versions 2.0.0 and above, although versions prior to 2.6.0
+will not report their full set of parameters. Agents will work regardless, but we may
+miss some configuration changes for older servers.
```
Comment (Preetam) [inline content/getting-started/supported-systems.md]: What about ElastiCache?

File: content/getting-started/supported-systems.md (hunk 1 above)
Comment (seaparks) [inline content/getting-started/supported-systems.md]: fake news!

let's see if this fixed it.

Comment (seaparks) [inline content/getting-started/supported-systems.md]: hm, that's odd.

**Pull request merged**